project/CI.scala

```diff
@@ -32,6 +30,7 @@ object CI {
                       |
                       |export IDEPROBE_DISPLAY=xvfb
                       |export JAVA_HOME=/usr/lib/jvm/java-11-openjdk
+                      |export IDEPROBE_SCREENSHOTS_DIR=/tmp/screenshots
```

Comment:
I think, according also to my first comment here, this should be `/tmp/ideprobe/screenshots`